commands/install.bee.inc

```diff
@@ -223,7 +223,11 @@ function install_bee_callback($arguments, $options) {
 
   // Run the install script.
   bee_render_text(array('value' => bt("\nInstalling Backdrop. This may take a moment...\n")));
-  $result = exec("cd $_bee_backdrop_root && ./core/scripts/install.sh $url $db $db_prefix $username $password $email $site_name $site_mail $langcode $clean_urls $profile");
+  // The core/scripts/install.sh binary is not installed as executable
+  // by default. Call it explicitly via the currently-specified PHP
+  // binary.
```

Comment:
A minor tweak. Coding standards allow up to 80 characters per line (https://docs.backdropcms.org/php-standards#line-length, https://docs.backdropcms.org/doc-standards#general) and this would benefit from that. Takes down to 2 lines.

```suggestion
  // The core/scripts/install.sh binary is not installed as executable by
  // default. Call it explicitly via the currently-specified PHP binary.
```